lib/ex_machina/ecto_strategy.ex

```diff
@@ -23,4 +25,71 @@ defmodule ExMachina.EctoStrategy do
   def handle_insert(_record, _opts) do
     raise "expected :repo to be given to ExMachina.EctoStrategy"
   end
+
+  defp cast(record) do
+    record
+    |> cast_fields
+    |> cast_assocs
+  end
+
+  defp cast_fields(struct) do
+    struct
+    |> ExMachina.Ecto.drop_ecto_fields
+    |> Map.keys
+    |> cast_fields(struct)
+  end
+
+  defp cast_fields(fields, struct) do
```

Comment:
Thoughts on naming this `cast_all_fields` or something like that? I didn't notice the `s` at the end and thought the next function was the same, but with different patterns. Changing the name might help to clarify that a bit